xamarin/Xamarin.Forms
# Dismiss UISearchController when losing focus

### Description of Change ###

Dismiss UISearchController when losing focus

### Issues Resolved ### 
- fixes #10124 

### API Changes ###
 None

### Platforms Affected ### 
- iOS

### Behavioral/Visual Changes ###
When a query is confirmed in SearchHandler, the search handler will lose focus and will no longer display a dimmed view on top of the remaining page

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###

1. Add a Shell SearchHandler
2. Tap the search bar to focus
3. Notice the main page is dimmed
4. Enter a search term
5. Execute the search
6. Notice the SearchHandler no longer displays a dimmed background and the main content is tappable

![out](https://user-images.githubusercontent.com/4793020/97806883-e10f9b80-1c2b-11eb-8191-aff2b403c826.gif)


### PR Checklist ###
